### DISCUSSION TITLE
Postpartum Resources for Dads

### DISCUSSION THREAD

**Discussion by BekahHW** [BekahHW/postpartum-wellness-app#70] [Resources]
Often, one of the first people with the potential to recognize postpartum issues is the dad. And at the same time, dads aren't provided the resources they need to identify postpartum issues and how they can be affected during the postpartum experience. In fact, there is evidence that suggests hormone shifts in dads in the postpartum period with 7-10% of dads also suffering from postpartum discussion.

Let's support dads with resources that can help them adjust to the new stage of life and help them to support the moms in their lives.

Drop your resources below with a quick description of how it will support dads. Please do your best to find reputable resources. 

This will go towards adding to [this issue](https://github.com/BekahHW/postpartum-wellness-app/issues/33).

**Comment by lmoroney** [comment]
What a wonderful idea, thanks! 

For me, having 2 children, and with mom suffering PPD quite severely, especially with the first one, I have a few pieces of advice.

1. Take her seriously. Don't make a joke about it. Sometimes it *is* funny, but make that something you can laugh about together, years later. 

2. It can happen at any time. Your strength is your patience and your endurance, not your muscles. Be kind. Swallow any frustration, and, much later it can be something you chat about with her to help grow your relationship.

3. Don't be afraid to signal that you are a partner in protecting mother and child, be be aware that you may also be smothering them. You probably have an instinct to protect, but realize expressing that instinct shouldn't be your first priority. Again, strength comes from within. 

4. A huge part of PPD is the mother fearing for the future. She feels vulnerable. Try to help her realize that she is safe with you, and you are there for her. 

5. My experience in PPD came from a desire to super-protect Mom and Daughter, and realizing that the world can be a huge an ugly place, and that I may not be able to cope with it. It was crushing. Realize that you don't have to do it all yourself.

6. I got so deep into (5,) that I would go into a deep deep sleep with my eyes wide open, and my baby daughter between my wife and I. Once, she woke up in the middle of the night and saw me like that, and panicked, thinking I had died. Imagine how that must have felt? It was only after that realization that I learned to try to let go, to relax, to be more confident that the baby will be fine. Mom will be fine, and the world is something that we can help shape, and not just react to. Don't get into the situation I was in! :)

7. One advice given by the Nurse as we left the hospital with our first baby: They don't come with a manual, but they're smart, and they will grow and learn. Give them love, care, and attention, and they'll be fine. There have been many billions of them in history. You aren't the first. In hindsight that really helped!
